ConnectionsPanel.js

```diff
@@ -420,4 +382,110 @@ ToolsButton.propTypes = {
   count:         PropTypes.number,
 };
 
+const ConnectionsPanelSection = ({ title, children }) => {
+  const { themeStr, interfaceLanguage } = useContext(GlobalStateContext);
+  const theme = getTheme(themeStr);
+  return (
+    <View style={styles.connectionPanelSection}  flexdir={interfaceLanguage === "english" ? null : styles.rtlRow}>
+      {title &&
+        <View style={styles.connectionPanelTitle}>
+          <Text style={[interfaceLanguage === "hebrew" ? styles.heInt : styles.enInt, theme.tertiaryText]}>
+            {title}
+          </Text>
+        </View>
+      }
+      <View>
```

Comment:
why is there an extra `<View>` here?

Reply:
i'm not sure i understand the question. i see here: 1. external View. 2. internal optional View for the title (with different styles). 3. internal View for the children

Reply:
Yeah, I got confused.